packages/environment-ember-loose/-private/intrinsics/get.d.ts

```diff
@@ -2,7 +2,7 @@ import { DirectInvokable, EmptyObject } from '@glint/template/-private/integrati
 import ObjectProxy from '@ember/object/proxy';
 
 export type GetHelper = DirectInvokable<{
-  <T, K extends keyof T>(args: EmptyObject, obj: T, key: K): T[K];
+  <T, K extends keyof T>(args: EmptyObject, obj: T | null | undefined, key: K): T[K];
```

Comment:
Thanks for the PR! I think we need two separate signatures for this, because `{{get maybeString 'length'}}` won't give me a `number` if `maybeString` is undefined.

Probably something like this?
```suggestion
  <T, K extends keyof T>(args: EmptyObject, obj: T, key: K): T[K];
  <T, K extends keyof T>(args: EmptyObject, obj: T | null | undefined, key: K): T[K] | undefined;
```

And then we should update the test accordingly to cover both cases.

Reply:
Done, I force-pushed the changes. Thanks for the help!